Makefile

```diff
@@ -112,9 +111,10 @@ generate-mocks:
 	GO111MODULE=on mockery -name '.*' -dir=engine/execution/computation/computer -case=underscore -output="./engine/execution/computation/computer/mock" -outpkg="mock"
 	GO111MODULE=on mockery -name '.*' -dir=engine/execution/state -case=underscore -output="./engine/execution/state/mock" -outpkg="mock"
 	GO111MODULE=on mockery -name '.*' -dir=fvm -case=underscore -output="./fvm/mock" -outpkg="mock"
-	GO111MODULE=on mockery -name '.*' -dir=network/gossip/libp2p/middleware -case=underscore -output="./network/gossip/libp2p/mock" -outpkg="mock"
-	GO111MODULE=on mockery -name 'Connector' -dir=network/gossip/libp2p -case=underscore -output="./network/gossip/libp2p/mock" -outpkg="mock"
-	GO111MODULE=on mockery -name 'SubscriptionManager' -dir=network/gossip/libp2p/channel -case=underscore -output="./network/gossip/libp2p/mock" -outpkg="mock"
+	GO111MODULE=on mockery -name '.*' -dir=network/p2p -case=underscore -output="./network/mock" -outpkg="mock"
```

Comment:
Out of scope here, but it might be a good idea to start outputting these mocks to packages named eg. `mocknetwork`. When they're all called `mock` we end up manually overwriting the package name for most tests, and not always consistentlyl.

Reply:
Addresses in https://github.com/onflow/flow-go/pull/183/commits/1009da8722661b25265e32022b25a17d367426c7